### DISCUSSION TITLE
about pos and pow

### DISCUSSION THREAD

**Discussion by Satya9-9** [Cyfrin/foundry-full-course-cu#1729] [General]
sir i am not getting the concept of pow and pos its confusing

**Comment by EngrPips** [comment]
Hello @Satya-1729, Both pos, and pow are sybil resistance mechanisms. 

**WHAT IS SYBIL RESISTANCE MECHANISM**
This mechanism prevents an individual from pretending to be a different number of people. For example, I said I wanted to give each person $1,000, and then you came to me ten times, putting on different costumes, which makes you look different each time, and you collected $10,000 instead of $1,000. You could do that because I didn't have a Sybil resistance mechanism. Assuming I said each person is entitled to $1,000, but they would verify with their fingerprint to get the money, and we would check that our system has not recorded their fingerprint in the past, it would become challenging to come more than once.

**HOW DOES POW WORKS AS SYBIL RESISTANCE MECHANISM**
In the Proof Of Work sybil resistance mechanism, you need to set up a machine that would continuously mine the blockchain to be entitled to the reward, so if you want to pretend to be different people, then you need to set up a different machine, which is more costly and not incentivizing to claim some little reward. In the POW sybil resistance mechanism, every machine races to see who will find the solution first, and whoever finds the solution first wins the block reward.

**HOW DOES POS WORKS AS SYBIL RESISTANCE MECHANISM**
In the Proof Of Stake, a Sybil resistance mechanism, validators stake their ETH to get a reward. In the POS sybil resistance system,, validators don't a race to solve a solution,,; instead, they are selected randomly to solve the blockchain problem. So, to be a validator, you need to stake some ETH, and if you want to pretend to be a different number of people, then you need to stake multiple ETH to do that, which isn't incentivizing  to claim some block reward.
